Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -982,7 +984,7 @@ static _WKAuthenticatorAttachment authenticatorAttachmentToWKAuthenticatorAttach
 - (void)makeCredentialWithChallenge:(NSData *)challenge origin:(NSString *)origin options:(_WKPublicKeyCredentialCreationOptions *)options completionHandler:(void (^)(_WKAuthenticatorAttestationResponse *, NSError *))handler
 {
 #if ENABLE(WEB_AUTHN)
-    auto clientDataJSON = produceClientDataJson(_WKWebAuthenticationTypeCreate, challenge, origin);
+    auto clientDataJSON = produceClientDataJson(_WKWebAuthenticationTypeCreate, challenge, origin, { }, false);
```

Comment:
default value for NSString* would read better as `nil` since you switched to NSString. Also, if you used default parameter values, we wouldn't need to update such call sites.